src/main/java/org/apache/maven/plugins/dependency/resolvers/OldResolveDependencySourcesMojo.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.maven.plugins.dependency.resolvers;
+
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+
+/**
+ * Goal that resolves the project source dependencies from the repository.
+ *
+ * @author <a href="mailto:brianf@apache.org">Brian Fox</a>
+ * @since 2.0-alpha2
+ * @deprecated in favor of {@code resolve-sources} goal and will be removed in a future version
```

Comment:
This message fails to explain that the goals are functionally identical, differing only in name. I looked through the plugin site and FAQ trying to find some explanation of why I should switch and what parameters I would need to change, finally coming across this PR via code archaeology.

Reply:
What difference does it make for that matter?

Reply:
I am just pointing out that when as a Maven user you see this warning in your build, you may spend time trying to figure out what all you need to do in order to migrate to the new goal, when in fact you need do nothing beyond add `resolve-` to your command.